website/docs/r/policy_set.html.markdown

```diff
@@ -49,6 +49,27 @@ resource "tfe_policy_set" "test" {
 }
 ```
 
+Local policies (non-VCS based usage, relies on Policy Set Version upload).
+
+_Note: this cannot be used when VCS is defined, and cannot be used when
+policy_ids are defined._
```

Comment:
This is true of the 'manually specified policies' one and the VCS one above, so looks weird here; I'd either omit it (this is a usage section and the schema will happily inform them of this anyway), or just do a single, general statement above all the examples saying that you choose one of these three methods.